pyproject.toml

```diff
@@ -28,11 +28,12 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.9,!=3.9.7"
-trulens-core = { version = "^1.0.0", extras = [
-  "snowflake",
-], allow-prereleases = true }
+trulens-core = { version = "^1.0.0", allow-prereleases = true }
 trulens-feedback = { version = "^1.0.0", allow-prereleases = true }
 trulens-dashboard = { version = "^1.0.0", allow-prereleases = true }
+trulens-instrument-basic = { version = "^1.0.0", allow-prereleases = true }
```

Comment:
Unsure if they need to be seperate packages. Is the problem with including them in trulens-core is that you'd like trulens-core to only contain the folder trulens/core and not trulens/instrument/... ?

Reply:
And if so, can we have "core instruments" in one package `trulens-instrument-core` and include all 3 in there?